src/System.IO.FileSystem/src/System.IO.FileSystem.csproj

```diff
@@ -20,6 +20,8 @@
     <Compile Include="System\IO\Enumeration\FileSystemEnumerableFactory.cs" />
     <Compile Include="System\IO\Enumeration\FileSystemEnumerator.cs" />
     <Compile Include="System\IO\Enumeration\FileSystemName.cs" />
+    <Compile Include="System\IO\FileSystem.DirectoryCreation.Windows.cs" />
```

Comment:
This is in the wrong section, it needs to be in the item group that is conditioned to windows.

Reply:
Oooh nice catch. I used VS to split it into Attributes + DirectoryCreation. Didn't notice the newly duplicated file got added to the wrong section.